### PR TITLE
Run systemtests as tekton pipeline

### PR DESCRIPTION
**Pull request by kornys**
### Type of change

- Enhancement / new feature

### Description

Ability to run enmasse systemtests as tekton pipeline on ocp4, this is only draft, it contains only fer params and it can test only master right now. But I will work more on that.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [x] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by lulf** [comment]
@kornys I think we should consider moving this out of systemtests either into a toplevel module (tekton or .tekton) where we can create other pipelines for building images as well. Also an additional option would be to put this in a separate repository.

**Comment by kornys** [comment]
@lulf yes I agree, we can move it i to .tekton and add there many others tasks and pipelines. I was thinking about ability to build you images, push into ocp registries and run systemtests. All steps with tekton

**Comment by lulf** [comment]
Anything missing here before we can merge it?

**Comment by kornys** [comment]
@lulf yes, I want to update that to use your current code from local repo and add ability to build images

**Comment by kornys** [comment]
@lulf I think it can be merged if you want, in current state it is able to run master code of systemtests, as next PR I will add more parameters, and off course pipeline which build images from branch

File: Makefile [hunk 1]
```diff
@@ -127,6 +127,9 @@ $(DOCKER_DIRS):
 systemtests:
 	make -C systemtests
 
+tekton-test:
```
Comment (lulf) [inline Makefile:130]: Maybe name it systemtests-tekton to make the names a bit more similar and logical?

Comment (kornys) [reply Makefile]: Renamed

**Pull request merged**